examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -16,19 +16,21 @@
 
 workspace = os.path.dirname(os.path.abspath(__file__))
 sys.path.append(workspace)
+sys.path.append("../../")
```

Comment:
use `root=os.path.dirname(workspace)` and then add `sys.path.append(root)` for code consistent